src/Hydra/Protocol/Figures/head-protocol-states.tex

```diff
@@ -1,5 +1,15 @@
 \begin{figure}[t!]
   \centering
+  \begin{tikzpicture}[>=stealth,auto,node distance=2.8cm, initial text=$\mathsf{deposit}$, every
+    state/.style={text width=10mm, text height=2mm, align=center}, color=red]
+    \node[state, initial, text width=12mm] (pending) {$\mathsf{pending}$};
+    \node[state] (final) [right of=pending] {$\mathsf{final}$};
+
+    \path[->] (pending) edge [bend right=20, below] node {$\mathsf{increment}$} (final);
+    \path[->] (pending) edge [bend left=20] node {$\mathsf{recover}$} (final);
```

Comment:
It is a bit misleading to call a final state after deposit _final_ since there is a final state in the bottom graph. Could we connect the two graphs somehow? Or have a separate increment graph?

Reply:
It's two state diagrams in one figure, yes. Both state diagrams have an initial edge and a final (terminal) state. Maybe I should indicate the "final" state to be just a terminal and not a literal state?

Reply:
Done in https://github.com/cardano-scaling/hydra-formal-specification/commit/2dcc790e4cb4569d684ab73c093939af98461a03